src/PermissionManagerInterface.php

```diff
@@ -31,6 +32,25 @@ public function getPermissionList($entity_type_id, $bundle_id);
    * @return array
    *   An array of permission names and descriptions.
    */
-  public function generateCrudPermissionList($group_content_entity_type_id, $group_content_bundle_id);
+  public function generateEntityOperationPermissionList($group_content_entity_type_id, $group_content_bundle_id);
+
+  /**
+   * Returns permissions that are enabled by default for the given role.
+   *
+   * @param string $group_entity_type_id
+   *   The entity type ID of the group for which to return permissions.
+   * @param string $group_bundle_id
+   *   The bundle ID of the group for which to return permissions.
+   * @param array $group_content_bundle_ids
```

Comment:
OK I remember why I put this in. Currently `getDefaultPermissions()` is the only 'entry point' to the `PermissionEvent` event subscriber. So this was the only place where I could pass the `$group_content_bundle_ids` to the event subscriber in the current code base.

This indeed doesn't make sense in this context. We only need to pass the group content bundle IDs when we create a new group content type. This is something we can do after #187 is in. We will then probably add a new method `getDefaultEntityOperationPermissions()` which we can use to populate the initial permissions for the new group content type.

I will remove it for now.